libgadget/hydra.c

```diff
@@ -292,12 +292,18 @@ hydro_ngbiter(
                   " We haven't implemented tracer particles and this shall not happen\n");
     }
 
+    /* Wind particles do not interact hydrodynamically: don't produce hydro acceleration
+     * or change the signalvel.*/
+    if(HYDRA_GET_PRIV(lv->tw)->WindOn && winds_is_particle_decoupled(other))
```

Comment:
Yes this makes sense -- if decoupled it doesn't make sense they still affect the signal velocity.